karaf/features/src/main/feature/feature.xml

```diff
@@ -27,6 +27,12 @@
 
     <!-- TODO how to add central.maven.org repo? -->
     <repository>http://central.maven.org/maven2/org/apache/jclouds/karaf/jclouds-karaf/1.9.2/jclouds-karaf-1.9.2-features.xml</repository>
+<!--
+TODO the above link works for loading it, at compile time, as in the file below is then installed into target/assembly/system/
+BUT the assembly's variant of *this* feature file keeps the link above which FAILS if the server is not online.
+it should contain the entry below *instead of* the entry above:
+    <repository>mvn:org.apache.jclouds.karaf/jclouds-karaf/1.9.2/xml/features</repository>
```

Comment:
The repository tag needs to contain a mvn URL (GAV format)
Few options 
1. Add maven central to `org.ops4j.pax.url.mvn.repositories.cfg` **NOTE** This will require internet access @ runtime so karaf can resolve it
2. Ensure that all jclouds dependencies are packaged in the karaf system directory @ compile (so internet access is not required)

I can't verify the above at the minute as my workspace is in a mess :smile: